client/packages/coldchain/src/common/BreachTypeCell.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import { RecordWithId, TemperatureBreachNodeType } from '@common/types';
+import {
+  CellProps,
+  SnowflakeIcon,
+  SunIcon,
+  Typography,
+  useTheme,
+  useTranslation,
+} from '@openmsupply-client/common';
+
+export const BreachTypeCell = <T extends RecordWithId>({
+  rowData,
+  column,
+}: CellProps<T>): ReactElement => {
+  const t = useTranslation('coldchain');
+  const theme = useTheme();
+
+  const breachType = column.accessor({
+    rowData,
+  }) as TemperatureBreachNodeType | null;
+  const temperature = breachType?.split('_')[0];
+  const type = breachType?.split('_')[1];
+
+  return (
+    <>
+      {breachType && (
+        <Typography
+          sx={{
+            color:
+              temperature === 'HOT'
+                ? theme.palette.warning.main
+                : theme.palette.secondary.dark,
```

Comment:
this really is just a suggestion.. an alternative to using `useTheme`
```suggestion
            color: temperature === 'HOT' ? 'warning.main' : 'secondary.dark',
```

and also

```
          {temperature === 'HOT' ? (
            <SunIcon sx={{ stroke: 'warning.main' }} />
          ) : (
            <SnowflakeIcon sx={{ stroke: 'secondary.dark' }} />
          )}
```

Reply:
Sorry, it wasn't clear - there's a two part suggestion here and the first part isn't resolved - and if you make that change, you can remove `useTheme` 😄